tutorials/introductory/usage.py

```diff
@@ -141,94 +137,78 @@
 #
 # and to convert a `np.matrix` ::
 #
-#   b = np.matrix([[1,2],[3,4]])
+#   b = np.matrix([[1, 2], [3, 4]])
 #   b_asarray = np.asarray(b)
 #
-# .. _pylab:
-#
-# Matplotlib, pyplot and pylab: how are they related?
-# ====================================================
+# .. _coding_styles:
 #
-# Matplotlib is the whole package and :mod:`matplotlib.pyplot` is a module in
-# Matplotlib.
+# The object-oriented interface and the pyplot interface
+# ======================================================
 #
-# For functions in the pyplot module, there is always a "current" figure and
-# axes (which is created automatically on request).  For example, in the
-# following example, the first call to ``plt.plot`` creates the axes, then
-# subsequent calls to ``plt.plot`` add additional lines on the same axes, and
-# ``plt.xlabel``, ``plt.ylabel``, ``plt.title`` and ``plt.legend`` set the
-# axes labels and title and add a legend.
+# As noted above, there are essentially two ways to use Matplotlib: one
+# can either explicitly create figures and axes, and call methods on them
```

Comment:
Would do this as a bullet list to give it visual structure:
~~~
# As noted above, there are essentially two ways to use Matplotlib:
# - The *object-oriented style* (OO-style): Explicitly create figures and axes, and call
#   methods on them.
# - Rely on *pyplot* to automatically create and manage the figures and axes and just
#   use the pyplot functions for plotting.

Reply:
done